wagn/lib/wagn/application.rb

```diff
@@ -71,11 +71,13 @@ def add_gem_path paths, path, options={}
     config.revisions_per_page    = 10
     config.request_logger        = false
 
+    # this needs to be on the application's paths object.
+    # maybe if we finally understand how these are supposed to be connected in railties we can fix lots of stuff
+    paths['db/migrate'] = Rails::Paths::Path.new(paths, 'db/migrate', "#{Cardio.gem_root}/db/migrate")
     paths = Decko::Engine.config.paths
+    paths['db/migrate'] = Rails::Paths::Path.new(paths, 'db/migrate', "#{Cardio.gem_root}/db/migrate")
     # should we have add_deck_paths for these?
     paths['local-mod'] = Rails::Paths::Path.new(paths, 'local-mod', "#{Rails.root}/mod") unless approot_is_gemroot?
```

Comment:
I may have missed it in a merge.  Or maybe there was a leftover that I kept.  So I should totally remove it?